app/mixins/table-map.js

```diff
@@ -150,14 +150,21 @@ export default Ember.Mixin.create({
       return columns;
     }
   }),
-  rankingsMap: [
-    { key: 'name' },
-    { key: 'year' },
-    { key: 'industry_eci' },
-    { key: 'eci' },
-    { key: 'gdp_pc_real' },
-    { key: 'gdp_real' },
-    { key: 'population' }
-   ],
+  rankingsMap: computed('featureToggle.showIndustries', function() {
+      let columns = [
+       { key: 'name' },
+       { key: 'year' },
+       { key: 'eci' },
+       { key: 'gdp_pc_real' },
+       { key: 'gdp_real' },
+       { key: 'population' }
+      ];
+
+    if(this.get('featureToggle.showIndustries')) {
+      columns.splice(3, 0, {key: 'industry_eci'});
```

Comment:
Hahaha, glad to know you still are looking at the PRs :P The splice feels a bit weird but I couldn't really figure out how to make that column conditionally appear and also have it show up in the right spot and not just the end of all the columns.